examples/orientation/orientation.ino

```diff
@@ -1,23 +1,83 @@
+ *                  Adafruit (https://github.com/adafruit). These libraries are
+ *                  mostly under an Apache License, Version 2.0.
+ *
+ *                  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *        Version:  1.0
+ *        Created:  10/29/2014
+ *       Revision:  none
+ *       Compiler:  Arduino
+ *
+ *         Author:  Ben Peters (ben@ardusat.com)
+ *   Organization:  Ardusat
+ *         Edited:  8/25/2015
+ *      Edited By:  Sam Olds (sam@ardusat.com)
+ *
+ * =====================================================================================
+ */
+
+/*-----------------------------------------------------------------------------
+ *  Includes
+ *-----------------------------------------------------------------------------*/
 #include <Arduino.h>
 #include <Wire.h>
 #include <ArdusatSDK.h>
 
+/*-----------------------------------------------------------------------------
+ *  Setup Software Serial to allow for both RF communication and USB communication
+ *    RX is digital pin 10 (connect to TX/DOUT of RF Device)
+ *    TX is digital pin 11 (connect to RX/DIN of RF Device)
+ *-----------------------------------------------------------------------------*/
+ArdusatSerial serialConnection(SERIAL_MODE_HARDWARE_AND_SOFTWARE, 8, 9);
+
+/*-----------------------------------------------------------------------------
+ *  Constant Definitions
+ *-----------------------------------------------------------------------------*/
+gyro_t orient;
+
+/*
+ * ===  FUNCTION  ======================================================================
+ *         Name:  setup
+ *  Description:  This function runs when the Arduino first turns on/resets. This is
+ *                our chance to take care of all one-time configuration tasks to get
+ *                the program ready to begin logging data.
+ * =====================================================================================
+ */
 void setup(void)
 {
-  Serial.begin(9600);
+  serialConnection.begin(9600);
 
   if (!beginGyroSensor()) {
-    Serial.println("can't init IMU");
+    serialConnection.println("Can't initialize IMU! Check your wiring.");
   }
+
+  /* We're ready to go! */
+  serialConnection.println("");
 }
 
+/*
+ * ===  FUNCTION  ======================================================================
+ *         Name:  loop
+ *  Description:  After setup runs, this loop function runs until the Arduino loses
+ *                power or resets. We go through and update each of the attached
+ *                sensors, write out the updated values in JSON format, then delay
+ *                before repeating the loop again.
+ * =====================================================================================
+ */
 void loop(void)
 {
-  gyro_t orient;
-
-  readGyro(&orient);
-
-  Serial.println(gyroToJSON("orientation", &orient));
+  readGyro(orient);
+  serialConnection.println(gyroToJSON("orientation", orient));
```

Comment:
let's change this from `"orientation"` to gyro - they're actually 2 completely distinct things, and this is measuring gyroscopic motion, not orientation.